javascript/stimulus_reflex.js

```diff
@@ -268,13 +273,13 @@ const setupDeclarativeReflexes = debounce(() => {
       const controllers = attributeValues(
         element.getAttribute(stimulusApplication.schema.controllerAttribute)
       )
-      const reflexes = attributeValues(
+      const reflexInstances = attributeValues(
```

Comment:
Should we rename this to something like `reflexAttributeNames`? It might be a little more accurate and explicit.

Reply:
Gods, I wish every problem was this easy to rectify.